src/coreclr/vm/eetwain.cpp

```diff
@@ -1189,6 +1189,21 @@ FCIMPL1(void, GCReporting::Register, GCFrame* frame)
 
     // Construct a GCFrame.
     _ASSERTE(frame != NULL);
+#ifdef FEATURE_INTERPRETER
+    Thread *pThread = GetThread();
+    Frame *pFrame = pThread->GetFrame();
+    frame->SetOSStackLocation(frame);
```

Comment:
This part can't be in the ifdef since you didn't ifdef guard some uses of GetOSStackLocation